### PR TITLE
Redo integration

### LINKED ISSUE #47: simplify integration tests

### PR DESCRIPTION
**Pull request by iricigor**
Main points from this PR:
- moved `storage account` and `identity` to script arguments (pipeline updated)
- tests grouped per command
- reused variables separated to multiple 

Future tasks that will be done in additional issues:
- #97 - split integration tests into multiple files (once we start with key vault things)
- #99 - testing documentation web page and generalize names/pipeline

Resolves #47 

Build ID [28579956](https://skype.visualstudio.com/SCC/_build/results?buildId=28579956&view=ms.vss-test-web.build-test-results-tab) with commit 96af026 (resolved conflict) completed successfully.

### CODE REVIEW

File: test/integration/integration_test.sh [hunk 1]
```diff
@@ -3,192 +3,175 @@
...
-  fi
-
-  if [ -f $PACKAGEFILE ]; then
-      test "$EVENT $PACKAGENAME package from file" assert.Success "dpkg -i $PACKAGEFILE"
-  else
-      test "$EVENT $PACKAGENAME package from repository" assert.Success "apt --assume-yes install $PACKAGENAME"
-  fi  
-  dpkg-query --showformat='  - ${Package} ${Version} installed\n' --show $PACKAGENAME
-}
+# calculated variables
```
Comment (MangyCoyote) [inline test/integration/integration_test.sh]: ?
Either add example or useless?

File: test/integration/integration_test.sh [hunk 2]
```diff
@@ -3,192 +3,175 @@
...
+
 testing start "$PACKAGENAME"
 testing class "package"
 test "Install fake package should fail" assert.Fail "apt --assume-yes install somenonexistingpackage"
 
 # dependencies installed?
 test "Check all dependencies are installed" assert.Success "dpkg -s libstdc++6"
 
-install_azmi
+test "Install $PACKAGENAME package from file" assert.Success "dpkg -i $PACKAGEFILE"
```
Comment (MangyCoyote) [inline test/integration/integration_test.sh]: Lost possible upgrade scenario.

Comment (iricigor) [reply test/integration/integration_test.sh]: Thanks for catching this. It's a topic I wanted to discuss. Taking it 1-1

File: test/integration/integration_test.sh (hunk 1 above)
Comment (iricigor) [reply test/integration/integration_test.sh]: I was playing with moving things around, and this comment remained. It's useless and now removed.

**Pull request merged**